cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -75,22 +71,21 @@ func (s *server) Start(_ context.Context, host component.Host) error {
 	// TODO OTel-collector does not initialize the tracer currently
 	// https://github.com/open-telemetry/opentelemetry-collector/issues/7532
 	//nolint
-	s.jtracer, err = jtracer.New("jaeger")
-	if err != nil {
-		return fmt.Errorf("could not initialize a tracer: %w", err)
-	}
 
+	internalTelset := telemetery.Setting{
```

Comment:
why not just `telset`?

Reply:
Because, telset is being passed as component.TelemeterySettings which is different from the server arg requirement of telemetery.Setting defined internally by us.

Reply:
there is no `telset` variable in scope, there's only `s.telset`, which does not conflict.

Reply:
Oh pardon, didn't understand that you were referencing to rename the internalTelset variable. Noted.